### PR TITLE
Shows upcoming passes with max elevation (and color coding)

### PR DESCRIPTION
**Pull request by tomaszmrugalski**
Here's an example result:

svarog-passes

This is still a WIP.

### CODE REVIEW

**Comment by tomaszmrugalski** [comment]
Timestamps calculations are off by 2 hours. Seems the code messes up local and UTC as everything is too early by 2h.

File: station/planner.py [hunk 1]
```diff
@@ -63,7 +122,11 @@ def execute(interval, cron=None):
 
     passes = get_passes(prediction_config, start, end)
     clear(cron)
```
Comment (fivitti) [inline station/planner.py:124]: The dry run clears the crontab. It shouldn't provide any changes.

**Review by fivitti — CHANGES_REQUESTED** [review]

File: station/planner.py (hunk 1 above)
Comment (tomaszmrugalski) [reply station/planner.py]: Ok, this is now fixed.

**Comment by tomaszmrugalski** [comment]
I've tried to address pylint complaints in the files I've touched, but it started to get messy. Let me know if you prefer to move those changes to a separate PR. Anyway, I hope this is now ready, @fivitti ;)

**Comment by tomaszmrugalski** [comment]
Hey, @fivitti, I've revived this, cleaned up, addressed issues, and rebased. CI is clean. OK to merge?

File: station/utils/globalvars.py [hunk 2]
```diff
@@ -2,8 +2,19 @@
...
+# environment, so it uses files in more convenient locations. Also,
+# it can enable maximum logging level.
+#
+# allowed cases
+# (not defined) => dev environment disabled
+# 1 => typcial use case, dev env enabled, max logging enabled
+# 2 => running from unit-tests, dev env enabled, logging on info,
+#      so tests are not too verbose
+
+DEV_ENVIRONMENT = os.environ.get("DEV_ENVIRONMENT") is not None
```
Comment (fivitti) [inline station/utils/globalvars.py:15]:
```suggestion
DEV_ENVIRONMENT = os.environ.get("DEV_ENVIRONMENT", "0") != "0"
```

**Review by fivitti — APPROVED** [review]

**Pull request merged**